src/examples/typescript/dapp/main.ts

```diff
@@ -289,6 +289,66 @@ async function hedera_signTransaction(_: Event) {
 document.getElementById('hedera_signTransaction')!.onsubmit = (e: SubmitEvent) =>
   showErrorOrSuccess(hedera_signTransaction, e)
 
+/**
+ * Methods used through DappSigner managing Transaction objects directly
+ */
+
+// 2. hedera_executeTransaction
+async function signer_hedera_executeTransaction(_: Event) {
+  const accountId = getState('signer-execute')
+
+  const bodyBytes = Buffer.from(getState('signer-execute-transaction-body'), 'base64')
+  const sigMap = base64StringToSignatureMap(
+    getState('signer-execute-transaction-signature-map'),
+  )
+
+  const bytes = proto.Transaction.encode({ bodyBytes, sigMap }).finish()
+  const transaction = Transaction.fromBytes(bytes)
+
+  const signer = dAppConnector.getSigner(AccountId.fromString(accountId))
```

Comment:
For example here, would it make more sense to do something like:

```
return dAppConnector.executeTransaction(transaction, AccountId.fromString(accountId))
```
The signer should only be used for the intended purpose and not a dumping ground for general walletconnect helper functions

Reply:
I think we could overload the functions in the dApp connector to provide this functionality there and not add additional methods to the signer implementation

Reply:
@hgraphql I think that sounds like a good idea, keeping the signer strictly related to the hedera signer implementation would be wise imo

Reply:
> For example here, would it make more sense to do something like:
> 
> ```
> return dAppConnector.executeTransaction(transaction, AccountId.fromString(accountId))
> ```
> 
> The signer should only be used for the intended purpose and not a dumping ground for general walletconnect helper functions

This feature was initially part of the last pull request; however, we opted to divide it into separate contributions for better clarity and management. I concur that incorporating this function through dappConnector is a viable option.

Moreover, integrating it within the signer facilitates the management of multiple signers. Developers have the flexibility to initialize and manage it independently from the dappConnector if they choose to do so.

Reply:
@franfernandez20 If they want to use it independently from the dapp connector then they should be using the included functions which cover all of these things already - signWithSigner, executeWithSigner, etc.

This seems to just be adding redundant functionality to the signer? It seems like most of this stuff is already handled inside the dappconnector so I'm not sure what exactly this is improving

Reply:
Using executeWithSigner, you are not invoking the defined method `hedera_executeTransaction`. I understand your perspective, and perhaps `hedera_signAndExecuteTransaction` could be used through the call method. The idea was to allow users to create a transaction easily and use it consistently, regardless of the specific method.